packages/react-server/core/__tests__/context/navigator/NavigatorRoutes.js

```diff
@@ -0,0 +1,34 @@
+import DumbPage from "./DumbPage";
+
+// This wrapper is taken from the compileClient function that writes the routes_(client|server).js file
+const pageWrapper = {
+	default: () => {
+		return {
+			done: (cb) => { cb(DumbPage); },
+		};
+	},
+};
+
+const Routes = {
+	"BasicPage": {
+		"path": ["/basicPage"],
+		"page": pageWrapper,
+	},
+	"BasicPageCaps": {
+		"path": ["/basicPageCaps"],
+		"page": pageWrapper,
+		"method": "GET", // this should be all caps because the req.getMethod() will return 'get'.
```

Comment:
This is testing that case sensitivity doesn't matter in the Routr package.  In versions <0.1.1, it did a case _sensitive_ match.  >=0.1.1, it added a `toLowerCase()` call first.  We want to make sure that doesn't regress.